be/src/exec/tablet_sink.cpp

```diff
@@ -121,7 +121,6 @@ void NodeChannel::open() {
         ptablet->set_tablet_id(tablet.tablet_id);
     }
     request.set_num_senders(_parent->_num_senders);
-    request.set_need_gen_rollup(_parent->_need_gen_rollup);
```

Comment:
We'd better add a comment in the protocol file to indicate the filed is deprecated.

Reply:
OK